css.md

```diff
@@ -0,0 +1,177 @@
+    font-family: sans-serif;
+    font-size: 16px;
+    font-style: italic;
+    font-weight: bold;
+    line-height: 1;
+    list-style: none;
+    text-align: right;
+}
+```
+
+_N.B.: It is not necessary to include the group headings in your actual CSS. They are just for clarity in this example._
+
+## Naming Conventions
+Custom [BEM ("Block Element Modifier")](https://en.bem.info/method/definitions/) format:
+
+```css
+.block-name
+.block-name_element-name
+.block-name__block-modifier
+.block-name_element-name__element-modifier
+```
+
+### Avoid creating elements of modifiers
+Appending an element name to a modifier class can result in a confusing class name like `.list__space_item`. 
+Avoid this in favor of using a descendant, like this: `.list__spaced .list_item`.
+
+### `id` attribute
+
+While the `id` attribute might be fine in HTML and JavaScript, it should be **avoided entirely** inside stylesheets for a few reasons:
+
+- ID selectors are not reusable
+- Can lead to priority issues
+
+##### Good
+
+```css
+.ur-name {
+  // ...
+}
+```
+
+##### Bad
+
+```css
+#ur-name {
+  // ...
+}
+```
+
+Just assign a class name to the element.
+
+## Property values
+
+These rules apply to your CSS property values
+
+- If the value of a property is `0`, do not specify units
+- The `!important` rule should be aggressively avoided.
+  - Keep style rules in a sensible order
+  - Compose styles to dissipate the need for an `!important` rule
+  - Fine to use in limited cases
+    - Overlays
+    - Declarations of the `display: none !important;` type
+- Use hex color codes `#000` unless there's an explicit need for an `rgba` declaration
+- Avoid mixing units
+- Unit-less `line-height` is preferred because it does not inherit a percentage value of its parent element, but instead is based on a multiplier of the `font-size`.
+
+##### Good
+
+```css
+.btn {
+  color: #222;
+}
+
+.btn-red {
+  color: #f00;
+}
+```
+
+##### Bad
+
+```css
+.btn-red {
+  color: #f00 !important;
+}
+
+.btn {
+  color: #222;
+}
+```
+
+## Media Queries
+In most cases styles should be declared mobile first, then enhanced with `min-width` media queries. 
+By doing this we create a base experience that all devices can use and one that does not require media query support.
+
+## Less
+
+At the CFPB we use Less as our CSS preprocessor.
+
+### Less General Practices
+
+- Organize your Less code into multiple files/modules
+- Put comments in `//` statements
+- Prefer nested selectors `.foo { .bar {} }` vs `.foo .bar {}`
+  - _Only if both `.foo` and `.foo .bar` need styling_
+- Don't nest selectors beyond 3 levels deep
+
+### Selectors and Nesting
+
+Styles shouldn't need to be nested more than three levels deep. This includes pseudo-selectors. 
+If you find yourself going further, think about re-organizing your rules _(either the specificity needed, or the layout of the nesting)_.
+
+### Variables
+
+- Colors: (use generic variable names like recent cfgov-refresh update)
```

Comment:
`like recent cfgov-refresh update` ... perhaps link to the update that's being referred to, or provide another example.